### PR TITLE
Cleaning up

### PR DESCRIPTION
**Pull request by bieniusa**
- Cleaning up of dependencies
- Replaced lager with logger
- Consistently using the logger API with macros now
- Adapated log levels for file and console logging
- Made filename and log levels configurable

### CODE REVIEW

**Comment by bieniusa** [comment]
Closes #2, #3, #1

File: examples/antidote_pb.config [hunk 1]
```diff
@@ -25,8 +25,8 @@
 %% Current operations: read, append, txn, update_only_txn, read_only_txn.
 %% antidote nodes. The assumption is that each node is assumed to have only one partition and the hash
 %% fun should be integer()%N).
-{operations, [{update_only_txn, 1}]}.
-%%{operations, [{read_only_txn, 1}]}.
+ {operations, [{update_only_txn, 1}]}.
```
Comment (albsch) [inline examples/antidote_pb.config]: Spacing after %% is inconsistent in this file

File: rebar3.crashdump [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Error: {badmatch,error}
```
Comment (albsch) [inline rebar3.crashdump]: Is this file needed?

File: script/connectDCs.erl [hunk 3]
```diff
@@ -404,7 +405,7 @@ get_external_read_dcs(OtherDCs) ->
 % Should return a list where each value is a sigle element tuple with the Dc number
```
Comment (albsch) [inline script/connectDCs.erl]: typo: sigle -> single
